Addressing #353 and #400 (#456)
The-Strategy-Unit/nhp_inputs: home.md

---
editor_options: 
  markdown: 

>     wrap: 72

Reviewer: Awkwardly, this editor setting has changed almost every line of this file so that they wrap at 72 characters.

> ---
> editor_options: 
>   markdown: 
>     wrap: 72
> ---
> 

Reviewer: Aha, this is just cruft and can be removed.
```suggestion
```

> ::: callout-note

Reviewer: This is a Markdown file, so callouts won't work here. The simplest thing to do is just remove the fences (so this will just appear as a standalone paragraph with no special markup).
```suggestion
```

**Please Note**: By accessing this model you are acknowledging that you
are aware of the implications of setting the model parameters, and have
completed any relevant training. If this is not the case, please contact
your Model Relationship Manager, or contact [the data science
team](mlcsu.su.datascience@nhs.net) before proceeding.

> :::

Reviewer: Removing callout fence as above.
```suggestion
```

Changing the provider will update all of the information shown within
the inputs application. You will immediately see the map update showing
the selected provider, as well as that trusts peers, as defined by the
[Trust Peer Finder
Tool](https://app.powerbi.com/view?r=eyJrIjoiMjdiOWQ4YTktNmNiNC00MmIwLThjNzktNWVmMmJmMzllNmViIiwidCI6IjUwZjYwNzFmLWJiZmUtNDAxYS04ODAzLTY3Mzc0OGU2MjllMiIsImMiOjh9).
The list of these peers is displayed below the map (collapsed by
default). Clicking on any of the points on the map will reveal the name
of the provider.

Once you have chosen a provider, you can pick from one of the baseline
years, and then select the year that you want to use as the model
horizon. This defaults to 15 years from the baseline.

Finally, you must specify a scenario name - this will be how the results
are identified in the inputs app.

> To run a model with your given parameters, you must contact [the data science
> team](mlcsu.su.datascience@nhs.net) who will add you to the relevant Posit Connect 
> Group to enable the 'Run Model' button to appear.

Reviewer: ```suggestion
To run a model with your given parameters, you must contact [the Data Science
team](mailto:mlcsu.su.datascience@nhs.net) who will add you to the relevant Posit Connect 
Group to enable the 'Run Model' button to appear.
```

Though actually, I wonder if we should encourage requests via MRMs so they can be vetted (and to protect the inbox).

Author: Not everyone will have an MRM though (e.g. regional users)?

Reviewer: My thinking here is that we (the inbox handlers) will have to do extra work to ask MRMs if a request is legit anyway. We could cut out that step if the request comes via the MRM in the first place.

And maybe everyone accessing the model _should_ have an MRM?

Additionally, the text here probably needs to note (again) that users must have completed their training before they can access the run model button. I realise you've added this to the note at the top, but I think a reminder is needed in this isolated bit of text.

Author: Have added the reiteration about training. Karen has requested we leave this as the DS inbox, as there is no 'MRM inbox' and the personnel who are MRMs may change over time.

## Evidence Map

Much evidence exists in published literature of the efficacy of various
methods to reduce or avoid acute hospital demand. Although difficult to

> summarise, this evidence has been mapped to provide an overview,

Reviewer: Might as well integrate the link into this para?

```suggestion
summarise, we have mapped this evidence in [an interactive app](https://connect.strategyunitwm.nhs.uk/nhp_evidence_map/) to provide an overview,
```

categorised by setting, evidence type, outcomes and effect. This map is
provided to enable exploration of the current evidence landscape,
identify gaps in knowledge, and allow searching of evidence by a range
of parameters.

> 
> To explore the interactive map, please [click
> here](https://connect.strategyunitwm.nhs.uk/nhp_evidence_map/).

Reviewer: Delete this link if integrating into the text above. 
```suggestion
```
